docs/overrides.rst

```diff
@@ -0,0 +1,167 @@
+that works across different sites**. The unique missing requirement is to be able to
+configure different Page Objects for each different site, because the extraction logic
+surely changes from site to site.
+This is exactly the functionality that overrides provides.
+
+Let's see is with an example. The following is a a spider that extract books
+from `books.toscrape.com <http://books.toscrape.com/>`_ using Page Objects.
+
+.. code-block:: python
+
+    import scrapy
+    from web_poet import ItemWebPage, WebPage
+    from scrapy_poet import callback_for
+
+
+    class BookListPage(WebPage):
+
+        def book_urls(self):
+            return self.css('.image_container a::attr(href)').getall()
+
+
+    class BookPage(ItemWebPage):
+
+        def to_item(self):
+            return {
+                'url': self.url,
+                'name': self.css("title::text").get(),
+            }
+
+
+    class BooksSpider(scrapy.Spider):
+        name = 'books_spider'
+        start_urls = ['http://books.toscrape.com/']
+
+        def parse(self, response, page: BookListPage):
+            for url in page.book_urls():
+                yield response.follow(url, callback_for(BookPage))
+
+Note that the crawling logic is pretty simple and straightforward:
+
+1. Extract all books urls from the listing page
+2. For each book URL found in the step 1, fetch the page and extract the resultant item
+
+This logic should work without any change for different books sites because
+having pages with lists of books and then detail pages for the individual book is
+such a common way of structuring sites.
+
+Let's reuse the very same ``BooksSpider`` spider but now to extract books from a different
+page: `bookpage.com/reviews <https://bookpage.com/reviews>`_.
+
+We cannot reuse the Books to Scrape Page Objects in this case, so we have
+to implement new ones:
+
+.. code-block:: python
+
+    class BPBookListPage(WebPage):
+
+        def book_urls(self):
+            return self.css('.article-info a::attr(href)').getall()
+
+
+    class BPBookPage(ItemWebPage):
+
+        def to_item(self):
+            return {
+                'url': self.url,
+                'name': self.css(".book-data h4::text").get().strip(),
+            }
+
+The last step is configuring ``scrapy-poet`` so that these Page Objects
+are used for Book Page URLs. This can be done by configuring
+``SCRAPY_POET_OVERRIDES`` in the Scrapy ``settings.py`` configuration file.
+
+.. code-block:: python
+
+    SCRAPY_POET_OVERRIDES = {
+        "bookpage.com": {
+            BookListPage: BPBookListPage,
+            BookPage: BPBookPage
+        }
+    }
+
+``SCRAPY_POET_OVERRIDES`` contain rules that overrides the Page Objects
+used for a particular domain. In this particular case, Page Objects
+``BPBookListPage`` and ``BPBookPage`` will be used instead of
+``BookListPage`` and ``BPBookPage`` for any request whose domain is
```

Comment:
`BPBookPage` -> `BookPage`

Reply:
Thank you! Fixed here: https://github.com/scrapinghub/scrapy-poet/pull/43/commits/eab5e92e0bb50e1da4302c938ad7f5cac058ef6e